android-timetracker/src/com/markuspage/android/atimetracker/Tasks.java

```diff
@@ -1072,12 +1102,15 @@ protected void onListItemClick(ListView l, View v, int position, long id) {
             }
         }
 
+        // Display the notification
+        
+        Object item = getListView().getItemAtPosition(position);
+            
         // Stop the update.  If a task is already running and we're stopping
         // the timer, it'll stay stopped.  If a task is already running and 
         // we're switching to a new task, or if nothing is running and we're
         // starting a new timer, then it'll be restarted.
 
-        Object item = getListView().getItemAtPosition(position);
```

Comment:
No, not really. I might have just jostled and added and deleted code and that just ended up moving.